src/reader.mjs

```diff
@@ -160,9 +156,8 @@ class ChunkedReader {
 
 
 class FsReader extends ChunkedReader {
```

Comment:
We can safely use `fs()` in `FsReader` because this is only ever used after an `isNode` check